backend/inspirehep/records/views.py

```diff
@@ -239,6 +247,40 @@ def reference_self_curation(args):
     return jsonify({"message": "Success"}), 200
 
 
+@blueprint.route("/literature/<path:recid>/<int:old_revision>..<int:new_revision>")
```

Comment:
you need either this or that https://github.com/inspirehep/inspirehep/pull/2905/files#diff-05fd21425f5249513d32ad78963077d2abad550c23b1a7e051efac7da7486d07R305-R308, because now it registers two endpoints in the app.  Doesn't it work if we just do the following:

```python
@blueprint.route("/literature/<<inspirepid(lit,record_class="inspirehep.records.api:LiteratureRecord"):pid_value>/<int:old_revision>..<int:new_revision>")
```